src/style/panel/participant-devices.less

```diff
@@ -19,28 +19,42 @@
 
 .participant-devices {
   &__fingerprint {
-    min-height: 72px;
+    width: 230px;
```

Comment:
do we really want such a fixed width? Seems like a magic number to me, what does this 230px correspond to?